packages/protocol/contracts/governance/Validators.sol

```diff
@@ -63,6 +63,8 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
 
   address constant PROOF_OF_POSSESSION = address(0xff - 4);
 
+  uint256 public maxGroupSize = 10;
```

Comment:
I don't think setting this here will do anything since we use proxied contracts.